data/migrations/20201013022647_create-groomer.js

```diff
@@ -2,19 +2,20 @@ exports.up = (knex) => {
   return knex.schema
     .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
     .createTable('Groomer', function (table) {
-      table.string('id').notNullable().unique().primary();
-      table.string('name').notNullable();
-      table.string('lastname').notNullable();
-      table.string('address').notNullable();
-      table.string('zip');
-      table.string('phone').notNullable().unique();
-      table.string('email').notNullable().unique();
-      table.string('city');
-      table.string('state');
-      table.string('country');
-      table.string('photo_url').unique();
-      table.integer('walk_rate');
-      table.integer('day_care_rate');
+      table.integer('id').notNullable().unique().primary();
```

Comment:
if you are changing to use INTs then also get rid of the uuid extension statement.

Reply:
You are right! I was considering using uuid and I'd need to use string as the type for "id".